cantools/db/attribute_definition.py

```diff
@@ -0,0 +1,98 @@
+class AttributeDefinition(object):
+    """A definition of an attribute that can be associated with attributes in nodes/messages/signals
+
+    """
+
+    def __init__(self,
+                 name,
+                 defaultValue=None,
+                 kind=None,
+                 type_=None,
+                 minimum=None,
+                 maximum=None,
+                 choices=None):
+        self._name = name
+        self._defaultValue = defaultValue
+        self._kind=kind
+        self._type_ = type_
+        self._minimum=minimum
+        self._maximum=maximum
+        self._choices = choices
+
+    @property
+    def name(self):
+        """The attribute name as a string.
+
+        """
+
+        return self._name
+
+    @property
+    def defaultValue(self):

```

Comment:
Use lower case with underscore, `default_value`. This applies to all methods, functions and variables.